MetaMask/metamask-extension
# Matchain develop

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27878?quickstart=1)

## **Related issues**

Fixes: Add Matchain logo svg

## **Manual testing steps**

Go to add network
Add Matchain
Matchain should have a logo
Native token should have a logo same with BSC

## **Screenshots/Recordings**



### **Before**
https://imgur.com/a/kZPYvMi


### **After**
https://imgur.com/a/Vmxa7KS


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
